graph/src/data/subgraph/schema.rs

```diff
@@ -362,3 +364,35 @@ fn set_entity_operation(
         data: data.into(),
     }
 }
+
+pub fn attribute_indexing_operations(
+    subgraph_id: SubgraphId,
+    document: Document,
+) -> Vec<AttributeIndexOperation> {
+    let mut indexing_ops = vec![];
+    for (entity_number, schema_type) in document.definitions.clone().into_iter().enumerate() {
+        if let Definition::TypeDefinition(definition) = schema_type {
+            if let TypeDefinition::Object(schema_object) = definition {
+                for (attribute_number, entity_field) in schema_object.fields.into_iter().enumerate()
+                {
+                    indexing_ops.push(AttributeIndexOperation {
+                        subgraph_id: subgraph_id.clone(),
+                        index_name: format!(
+                            "{}_{}_{}_idx",
```

Comment:
Do we even need the `_idx` suffix?

Reply:
I included this to follow Postgres naming conventions as referenced here, https://gist.github.com/popravich/d6816ef1653329fb1745.
It is not completely necessary; when looking at the DB it should be clear to Graph developers that these relations are indexes, but it's always nice to be specific just in case. :)

Reply:
Ok, agreed.